temporalcli/commandsgen/commands.yml

```diff
@@ -3333,3 +3304,37 @@ option-sets:
         description: |
           Run ID.
           If unset, updates the currently-running Workflow Execution.
+
+  - name: nexus-endpoint-identity
+    description: Nexus endpoint commands
+    options:
+      - name: name
+        type: string
+        description: Endpoint name.
+        required: true
+
+  - name: nexus-endpoint-config
+    description: Nexus endpoint commands
+    options:
+      - name: description
+        type: string
+        description: |
+          Nexus Endpoint description.
+          You may use Markdown formatting in the Nexus Endpoint description.
```

Comment:
Does the description need to be in JSON string format, that is `"**Sales Services**: Workflows to support Customer-to-Order generation"`? If so, recommend example.

(Also, won't that be a pain with Unix expansions for some items?)

Reply:
no just a string will do, [see comment below](https://github.com/temporalio/cli/pull/693/files#r1803571528)

Reply:
Does not require JSON string formatting, marking as resolved.